docs/en/features/local_census.md

```diff
@@ -1,33 +1,36 @@
-# Local Census
+# Local census
 
-To provide to administrator users a way to manage the local census database through the administration panel **Settings > Manage local census**. Currently the only way to manipulate this table records is through the rails console.
+This feature is designed for installations with limited resources that cannot configure and customize their census connection remotely, so we offer them a solution to use a local census instead. For this purpose, administrators are provided with a way to manage the local census database through the administration panel under **Settings > Local census**.
 
-Allow administrators users to manage this table in two different ways:
+## Manage local census
 
-- **Manually**: one by one through a CRUD interface.
-- **Automatically**: through an importation process.
+Administrators can manage this census in two different ways:
 
-## Manually
+* **Manually**: add one by one through a form.
+* **Automatically**: through an importation process.
 
-Provide a way to manage local census records to administrator users through administration interface.
+### Manually
 
-- Local Census Page
-  ![Manage local census](../../img/local_census/manage-local-census-en.png)
+It allows administrators to create a record by clicking the _Create new local census record_ button, which appears in the top right corner, as shown in the following image:
 
-- Add new record
-  ![Create local census record](../../img/local_census/add-local-census-record-en.png)
+![Create a record in the local census button](../../img/local_census/manage-local-census-en.png)
 
-Features:
+This will take us to a new page where we can fill in the following form and create the new record:
 
-1. Search by document_number: As local_census_records could contain a lot of records we have added a search feature to allow administrators to find existing records by document_number.
-1. Avoid the introduction of duplicated records: A model validation has been added to the following attributes pair [:document_number, :document_type]
+![Form to create a record](../../img/local_census/add-local-census-record-en.png)
```

Comment:
Since we don't mention the fields present in the form anywhere else, maybe we could write something a bit more descriptive :thinking:.

```suggestion
![Form to create a record by filling in the document type, document number, date of birth and postal code](../../img/local_census/add-local-census-record-en.png)
```